.github/workflows/inspektor-gadget.yml

```diff
@@ -416,6 +416,39 @@ jobs:
               ${{ steps.set-repo-determine-image-tag.outputs.container-repo }}@${{ needs.build-gadget-container-images.outputs.digest-default-arm64 }}
           fi
 
+  build-helper-images:
+    name: Build helper images
+    # level: 0
+    if: github.repository == 'inspektor-gadget/inspektor-gadget'
+    runs-on: ubuntu-latest
+    permissions:
+      # allow publishing container image
+      # in case of public fork repo/packages permissions will always be read
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        image: [ {dockerfile-dir: "tools/dnstester",  name: "dnstester", tag: "latest"} ]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Login to Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Build ${{ matrix.image.name }} image
+      uses: docker/build-push-action@v4
+      with:
+        context: ${{ matrix.image.dockerfile-dir }}
+        file: ${{ matrix.image.dockerfile-dir }}/Dockerfile
+        push: true
+        tags: ghcr.io/inspektor-gadget/${{ matrix.image.name }}:${{ matrix.image.tag }}
+        platforms: linux/amd64,linux/arm64
```

Comment:
nit: probably this should be also in the matrix as I suspect we're going to push other images that don't need to be multiplatform using this action in the future, like ebpf-builder.